package.json

```diff
@@ -19,14 +17,11 @@
     "package-plugin": "./bin/build-plugin-zip.sh"
   },
   "devDependencies": {
+    "@wordpress/babel-preset-default": "^1.1.0",
+    "@wordpress/browserslist-config": "^2.1.0",
```

Comment:
This should not be required as it is included in the @wordpress/babel-preset-default package https://github.com/WordPress/packages/blob/master/packages/babel-preset-default/package.json#L25